app/views/member_mailer/welcome_student.html.haml

```diff
@@ -22,7 +22,7 @@
   you to a coach with one to three other students. You can either work through
   = link_to "one of our tutorials", "http://tutorials.codebar.io/"
   or get help with your own project. After 2 hours of coaching, we retire to a nearby
-  pub for some post-Codebar relaxation.
+  pub for some post-codebar relaxation.
```

Comment:
No, this text needs to be reviewed. The pub is not a requirement for running codebar and it shouldn't be included here.